### PR TITLE
fix: textInput label wrong position in RTL

### LINKED ISSUE #562: TextInput with RTL is buggy
Saw this now on Expo (pixel 2 xl, android pie, hebrew)
screenshot_20180930-012528

### PR DESCRIPTION
**Pull request by iyadthayyil**
### Motivation

Fixes #562 
whatsapp image 2018-09-30 at 4 20 28 pm 1
whatsapp image 2018-09-30 at 4 20 28 pm

### CODE REVIEW

**Comment by iyadthayyil** [comment]
Fixed screenshot:

whatsapp image 2018-09-30 at 10 06 04 pm

File: src/components/TextInput.js [hunk 1]
```diff
@@ -448,6 +448,16 @@ class TextInput extends React.Component<Props, State> {
             ],
           }),
         },
+        {
+          // RTL Support
+          translateX: this.state.labeled.interpolate({
+            inputRange: [0, 1],
+            outputRange: [
+              I18nManager.isRTL ? (20 / 100) * this.state.labelLayout.width : 0,
```
Comment (ferrannp) [inline src/components/TextInput.js]: Can you explain where these numbers come from?

**Comment by saharjedidi** [comment]
@iyadthayyil  Please how can I fix the label position in RTL Mode 
error
this is my code 
<TextInput
            mode='outlined'
            label=' رقم بطاقة التعريف الوطنية'
            numberOfLines={1}
            style={this.getStyle().TextInp}
            theme={{ colors: {placeholder: 'white', background:'#003489',text: 'white',primary: 'white'}}}
            onChangeText={CIN => this.setState({CIN})}
            onBlur={ this.onBlurCIN }
            />

**Comment by Trancever** [comment]
@saharjedidi Did you try this branch? Currently there is a bug on master branch

**Comment by saharjedid** [comment]
@Trancever my version is  "react-native-paper": "^2.2.4",
did you know what can i add in my code to fix the problem 
i didn't find any documentation about this problem

**Comment by Trancever** [comment]
Closing this in favour of https://github.com/callstack/react-native-paper/pull/721

**Pull request closed without merging**